README.md

```diff
@@ -183,18 +231,37 @@ several JIRA installations from one piece of code. For example, if you want to w
 instances at the same time:
 
 ```php
-$Prod = new \Badoo\Jira\REST\Client('https://jira.example.com/');
-$Prod->setAuth('user', 'password/token');
+$Prod = new \Badoo\Jira\REST\Client(new \Badoo\Jira\REST\ClientRaw('https://jira.example.com/'));
+$Prod->setAuth('produser', 'password/token');
 
-$Staging = new \Badoo\Jira\REST\Client('https://staging.jira.example.com/');
-$Staging->setAuth('user', 'password/token');
+$Staging = new \Badoo\Jira\REST\Client(new \Badoo\Jira\REST\ClientRaw('https://jira.example.com/'));
+$Staging->setAuth('staginguser', 'password/token');
 
 $ProdIssue = new \Badoo\Jira\Issue('SMPL-1', $Prod);
 $StagingIssue = new \Badoo\Jira\Issue('SMPL-1', $Staging);
 
 // ...
 ```
 
+#### The simplest interface to API: \Badoo\Jira\REST\ClientRaw
+
+It can request API and parse responses.
+Throws an `\Badoo\Jira\REST\Exception` for API errors or parsed response data when everything went OK.
```

Comment:
I'm not an English master, but I think there is something wrong with this sentence. I guess you meant something like: "Parses response data. On API error, throws `\Badoo\Jira\REST\Exception`."